catalog/queries/search/muziekschatten-onderwerpen.rq

```diff
@@ -14,8 +14,7 @@ WHERE {
     ?uri ?predicate ?label .
     VALUES ?predicate { schema:name schema:keywords }
 
-    # A 'FILTER (<bif:contains>, ...)' currently does not work due to https://github.com/comunica/comunica/issues/844
-    ?label <bif:contains> ?query .
+    ?label <bif:contains> ?booleanQuery .
```

Comment:
Our Virtuoso queries take two forms: `FILTER (<bif:contains> (?label, ?booleanQuery))` and `?label <bif:contains> ?booleanQuery`. Would it make sense to use one form instead?

Reply:
Yes! It says [here](http://docs.openlinksw.com/virtuoso/rdfsparqlrulefulltext/#rdfsparqlrulespecifywhatindexexample) that these two queries are identical and differ in syntax only:

```sparql
?o bif:contains '"Timo*"'
```

is equivalent to

```sparql
FILTER (bif:contains(?o,  '"Timo*"')
```

> The filter syntax is more flexible in that it allows passing extra options to the contains predicate. These may be useful in the future.

Because we don't need to supply any options (right now), we could use both syntaxes. `?o bif:contains '"Timo*"'` is perhaps a bit easier on the eyes, so I pick that one. Which syntax do you prefer?